src/types/onyx/Policy.ts

```diff
@@ -1267,6 +1267,9 @@ type PolicyReportField = {
     /** Value of the field */
     value?: string | null;
 
+    /** Value of the target */
+    target?: string;
```

Comment:
I think we could be more strict about this. Target can only be `expense|invoice|paycheck` coming from a back end

Reply:
Updated! 😄